contrib/utilities/indent-branch

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2018 by the deal.II authors
+##
+## This file is part of the deal.II library.
+##
+## The deal.II library is free software; you can use it, redistribute
+## it, and/or modify it under the terms of the GNU Lesser General
+## Public License as published by the Free Software Foundation; either
+## version 2.1 of the License, or (at your option) any later version.
+## The full text of the license can be found in the file LICENSE at
+## the top level of the deal.II distribution.
+##
+## ---------------------------------------------------------------------
+
+#
+# This script does the same thing as contrib/utilities/indent but only reformats
+# files which have changed (or been added and not committed) since the last
+# commit to the master branch. It should be run as
+#
+#    make indent-branch
+#
+# from a build directory.
+#
+
+if test ! -d source -o ! -d include -o ! -d examples ; then
```

Comment:
All of this duplicated code from `indent`, right? If so, maybe put all of the common code into a file `indent.common` and then just do `source indent.common` here and in `indent`?

Reply:
Yes. Do you mind if I name it `indent-checks`? I prefer to avoid `.`s unless we have filename extensions.

Reply:
Yes, any name is ok with me.